wsgiservice_restplus/fields.py

```diff
@@ -92,8 +92,8 @@ class Raw(object):
         than the publicly named value.
     :param str title: The field title (for documentation purpose)
     :param str description: The field description (for documentation purpose)
-    :param bool required: Is the field required ?
-    :param bool readonly: Is the field read only ? (for documentation purpose)
+    :param bool required: Is the field mandatory?
```

Comment:
On the same page with you on that one, however, after a guick inspection I'd suggest we rather don't do it at this point, as a lot of code elsewhere uses the field.required attribute and it could be tricky to hunt all of them (Pycharm did not pick them up for example)